### PR TITLE
continuous logging added to multimeter

### LINKED ISSUE #1762: Allow user to set time interval to continuously log data in multimeter
**Actual Behaviour**
Currently user has to press READ button continuously to record the log

**Expected Behaviour**
Once user set a time interval. After the record is pressed data will automatically logged for the entity to which the knob is pointing so the user doesn't have to continuously press READ button

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by neel1998**
Fixes #1762 #1373 

**Changes**: User can now continuously log data in multimeter and can set time interval in configuration settings. Reset / Read buttons removed

**Screenshot/s for the changes**:

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[multimeter.zip](https://github.com/fossasia/pslab-android/files/3244573/multimeter.zip)

@CloudyPadmal  @ho-dor  please review

### CODE REVIEW

**Comment by mariobehling** [comment]
Could you please get rid of the "read" and "reset" button and implement it in a way that it always shows the data input, if any. In a multimeter device there is no such thing as a read or input button. As soon as the user connects the device to do measurements it shows the output. Same should happen here. Please implement it in the same way like an actual multimeter.

**Comment by neel1998** [comment]
@mariobehling  on it. :)

**Comment by neel1998** [comment]
@mariobehling  i have made the changes you suggested. Please have a look

File: app/src/main/res/layout-hdpi/activity_multimeter.xml [hunk 1]
```diff
@@ -140,11 +140,11 @@
...
                 android:layout_width="match_parent"
                 android:layout_height="@dimen/multimeter_line_width"
-                android:layout_marginBottom="16dp"
-                android:layout_marginEnd="@dimen/multimeter_constraint_1"
+                android:layout_marginStart="@dimen/multimeter_constraint_1"
                 android:layout_marginLeft="@dimen/multimeter_constraint_1"
+                android:layout_marginEnd="@dimen/multimeter_constraint_1"
                 android:layout_marginRight="@dimen/multimeter_constraint_1"
-                android:layout_marginStart="@dimen/multimeter_constraint_1"
+                android:layout_marginBottom="16dp"
```
Comment (ho-dor) [inline app/src/main/res/layout-hdpi/activity_multimeter.xml]: Hard coded dimension

File: app/src/main/res/layout-hdpi/activity_multimeter.xml [hunk 2]
```diff
@@ -216,11 +216,11 @@
...
                 android:layout_width="@dimen/multimeter_length_0"
                 android:layout_height="wrap_content"
-                android:layout_marginBottom="5dp"
-                android:layout_marginEnd="@dimen/multimeter_constraint_2"
+                android:layout_marginStart="@dimen/multimeter_constraint_2"
                 android:layout_marginLeft="@dimen/multimeter_constraint_2"
+                android:layout_marginEnd="@dimen/multimeter_constraint_2"
                 android:layout_marginRight="@dimen/multimeter_constraint_2"
-                android:layout_marginStart="@dimen/multimeter_constraint_2"
+                android:layout_marginBottom="5dp"
```
Comment (ho-dor) [inline app/src/main/res/layout-hdpi/activity_multimeter.xml]: Here as well

File: app/src/main/res/layout-hdpi/activity_multimeter.xml [hunk 3]
```diff
@@ -412,11 +412,11 @@
...
                 android:layout_width="@dimen/multimeter_length_0"
                 android:layout_height="wrap_content"
-                android:layout_marginBottom="8dp"
-                android:layout_marginEnd="@dimen/multimeter_constraint_1"
+                android:layout_marginStart="@dimen/multimeter_constraint_1"
                 android:layout_marginLeft="@dimen/multimeter_constraint_1"
+                android:layout_marginEnd="@dimen/multimeter_constraint_1"
                 android:layout_marginRight="@dimen/multimeter_constraint_1"
-                android:layout_marginStart="@dimen/multimeter_constraint_1"
+                android:layout_marginBottom="8dp"
```
Comment (ho-dor) [inline app/src/main/res/layout-hdpi/activity_multimeter.xml]: Here too

File: app/src/main/res/layout-hdpi/activity_multimeter.xml [hunk 4]
```diff
@@ -412,11 +412,11 @@
...
                 android:gravity="center"
                 android:text="@string/multimeter_measure"
                 android:textSize="14sp"
```
Comment (ho-dor) [inline app/src/main/res/layout-hdpi/activity_multimeter.xml]: Change this to a dimen resource.

File: app/src/main/res/layout-xhdpi/activity_multimeter.xml [hunk 5]
```diff
@@ -24,19 +24,19 @@
     <android.support.constraint.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="0dp"
-        android:layout_marginBottom="8dp"
         android:layout_marginTop="8dp"
+        android:layout_marginBottom="8dp"
```
Comment (ho-dor) [inline app/src/main/res/layout-xhdpi/activity_multimeter.xml]: Same issue

File: app/src/main/res/layout-xhdpi/activity_multimeter.xml [hunk 6]
```diff
@@ -85,9 +85,9 @@
         <android.support.constraint.ConstraintLayout
             android:layout_width="match_parent"
             android:layout_height="0dp"
-            android:layout_marginBottom="8dp"
             android:layout_marginTop="5dp"
-            app:layout_constraintBottom_toTopOf="@+id/reset"
+            android:layout_marginBottom="8dp"
```
Comment (ho-dor) [inline app/src/main/res/layout-xhdpi/activity_multimeter.xml]: Same

File: app/src/main/res/layout-xhdpi/activity_multimeter.xml [hunk 7]
```diff
@@ -141,11 +141,11 @@
...
                 android:layout_width="match_parent"
                 android:layout_height="@dimen/multimeter_line_width"
-                android:layout_marginBottom="24dp"
-                android:layout_marginEnd="@dimen/multimeter_constraint_1"
+                android:layout_marginStart="@dimen/multimeter_constraint_1"
                 android:layout_marginLeft="@dimen/multimeter_constraint_1"
+                android:layout_marginEnd="@dimen/multimeter_constraint_1"
                 android:layout_marginRight="@dimen/multimeter_constraint_1"
-                android:layout_marginStart="@dimen/multimeter_constraint_1"
+                android:layout_marginBottom="24dp"
```
Comment (ho-dor) [inline app/src/main/res/layout-xhdpi/activity_multimeter.xml]: Check please. Same issue

File: app/src/main/res/layout-xhdpi/activity_multimeter.xml [hunk 8]
```diff
@@ -155,11 +155,11 @@
...
                 android:layout_width="@dimen/multimeter_length_0"
                 android:layout_height="@dimen/multimeter_length_0"
-                android:layout_marginBottom="8dp"
-                android:layout_marginEnd="@dimen/multimeter_left_box_endmargin"
+                android:layout_marginStart="@dimen/multimeter_layout_margin_2"
                 android:layout_marginLeft="@dimen/multimeter_layout_margin_2"
+                android:layout_marginEnd="@dimen/multimeter_left_box_endmargin"
                 android:layout_marginRight="@dimen/multimeter_left_box_endmargin"
-                android:layout_marginStart="@dimen/multimeter_layout_margin_2"
+                android:layout_marginBottom="8dp"
```
Comment (ho-dor) [inline app/src/main/res/layout-xhdpi/activity_multimeter.xml]: Same

**Comment by neel1998** [comment]
@ho-dor @CloudyPadmal  I removed the hard coded dimensions. Please review

**Pull request merged**